authenticate-users/index.php

```diff
@@ -94,7 +94,7 @@ function validate_assertion(string $assertion, string $certs, string $audience)
             printf("<h1>Hello %s</h1>", $email);
         }
         break;
-    case ''; break; // Nothing to do, we're running our tests
+    case '': break; // Nothing to do, we're running our tests
```

Comment:
non-blocking: Surprised this clears style check.

Reply:
It doesn't, I found a bug where the cs check wasn't failing the test when it failed (it's now fixed).